README.md

```diff
@@ -0,0 +1,43 @@
+# CmClickpost
+
+Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/cm_clickpost`. To experiment with that code, run `bin/console` for an interactive prompt.
+
+TODO: Delete this and the text above, and describe your gem
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'cm_clickpost'
+```
+
+And then execute:
+
+    $ bundle install
+
+Or install it yourself as:
+
+    $ gem install cm_clickpost
+
+## Usage
+
+TODO: Write usage instructions here
```

Comment:
@jish0desarkar - we'll need to get this done ASAP :)

Reply:
@jish0desarkar This needs to be update